frontend/packages/@depmap/data-explorer-2/src/services/deprecatedDataExplorerAPI/portalMethods.ts

```diff
@@ -148,6 +151,21 @@ export async function fetchLegacyAssociations(
 export async function evaluateLegacyContext(
   context: DataExplorerContext | DataExplorerAnonymousContext
 ): Promise<string[]> {
+  // It's possible that we've been passed a V2 context
+  // because this hacky wrapper is not properly typed:
+  // https://github.com/broadinstitute/depmap-portal/blob/5b38776/frontend/packages/@depmap/data-explorer-2/src/components/ContextSelector/index.tsx#L6-L15
```

Comment:
(always uses V1 props even when V2 is returned)
https://github.com/broadinstitute/depmap-portal/blob/5b38776201e06a104f139d12f4196a900e9f238b/frontend/packages/@depmap/data-explorer-2/src/components/ContextSelector/index.tsx#L6-L15